src/equiv.js

```diff
@@ -12,8 +12,11 @@ QUnit.equiv = (function() {
 	function useStrictEquality( b, a ) {
 
 		/*jshint eqeqeq:false */
-		if ( b instanceof a.constructor || a instanceof b.constructor ) {
+		if ( b instanceof a.constructor && a instanceof b.constructor ) {
 
+			// Compare same constructor.
+			return a.toString() == b.toString();
```

Comment:
Well, there are two issues. First, and most importantly, we should probably have an automatic `return true` in a separate `guaranteed` function for `null`, `undefined`, and `nan` "types". Having cleared those out of the way, there would be a few options for detecting wrapped primitives, and I like `typeof object` because it strikes me as the most semantic. And separate from _all_ of that, it's just nice to see a single `return` with uniform comparison logic.